app/assets/stylesheets/pages/issues.scss

```diff
@@ -4,14 +4,23 @@
     position: relative;
 
     .issue-title {
-      margin-bottom: 5px;
       font-size: $list-font-size;
+      margin-bottom: 5px;
       font-weight: bold;
+      float: left;
+      width: 97.7%;
+    }
+    
+    @media screen and (max-width: 1100px){
+      .issues-list .issue .issue-title{width: 97%}
```

Comment:
Please format this style like the others, on its own line, and include a space before the opening `{` for both the selector and the media query.